nodes/persistence-node/src/services/IpfsGatewayApi.ts

```diff
@@ -114,20 +104,30 @@ export class IpfsGatewayApi {
       })
     }));
 
-    app.get('/ipfs/:hash', handleError(async (req, res) => {
+    app.get("/ipfs/:hash(*)", handleError(async (req, res) => {
       const hash = (req.params as any).hash as string;
```

Comment:
its an ipfsPath, not a hash, it can be e.g. ""Qm......sd/meta/schema.graphql"

Reply:
Fixed.